model/nc_news-model.js

```diff
@@ -21,4 +20,30 @@ exports.selectArticleByID = (article_id) => {
         }
         return result.rows[0]
     })
-}
\ No newline at end of file
+}
+exports.selectAllArticles = () => {
+    return db.query(`
+     SELECT articles.*, COUNT(articles.article_id) AS comment_count
+     FROM articles 
+     LEFT JOIN comments 
+     ON articles.article_id = comments.article_id 
+     GROUP BY articles.article_id 
+     ORDER BY created_at DESC
+     ;`).then((result) => {
+       const articlesModified = result.rows;
+       articlesModified.forEach((article)=>{ 
+        delete article.body;
+       })
+       return articlesModified
+    }).catch((err)=>{
+        if(err) {
+            console.log(err)
```

Comment:
Why would you console log an error here? What error could possibly happen on a select from a table that won't be empty?

Reply:
I just forgot that haha